addons/mrp/static/src/widgets/mrp_consumed.js

```diff
@@ -0,0 +1,47 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+import { FloatField } from '@web/views/fields/float/float_field';
+
+const { useRef, useEffect } = owl;
+
+export class MrpConsumed extends FloatField {
+    setup() {
+        super.setup();
+        const inputRef = useRef("numpadDecimal");
+        const readonlyRef = useRef("readonly");
+
+        useEffect(
+            (inputEl) => {
+                if (inputEl) {
+                    inputEl.addEventListener("input", this.onInput.bind(this));
+                    return () => {
+                        inputEl.removeEventListener("input", this.onInput.bind(this));
+                    };
+                }
+            },
+            () => [inputRef.el]
+        );
+
+        useEffect(
+            (readonlyEl) => {
+                if (readonlyEl) {
+                    if (this.props.record.data.manual_consumption) {
+                        readonlyEl.parentElement.parentElement.classList.remove('o_non_manual_consumption');
+                    } else {
+                        readonlyEl.parentElement.parentElement.classList.add('o_non_manual_consumption');
+                    }
+                }
+            },
+            () => [readonlyRef.el]
+        );
```

Comment:
Can't do it in this widget, since the cell I want to change is part of the list not the widget. 
to make it better, I do this part in the list renderer